clients/web/src/templates/body/systemConfiguration.jade

```diff
@@ -46,7 +74,7 @@ form.g-settings-form(role="form")
       option(value="yesmod", selected=(settings['core.add_to_group_policy'] ==
              "yesmod")) Yes by group administrators and moderators unless disabled per group
 
-  .g-collection-create-policy-container
+  .g-settings-form-container
```

Comment:
Ah, that's cool, it should work if you keep the old class also. Classes are, to some degree, part of our API surface.